lib/containers/common.pm

```diff
@@ -25,7 +25,7 @@ use utils qw(zypper_call systemctl file_content_replace script_retry);
 use version_utils qw(is_sle is_leap is_microos is_sle_micro is_opensuse is_jeos is_public_cloud get_os_release check_version);
 use containers::utils qw(can_build_sle_base registry_url);
 
-our @EXPORT = qw(install_podman_when_needed install_docker_when_needed allow_selected_insecure_registries
+our @EXPORT = qw(install_podman_when_needed install_docker_when_needed
   clean_container_host test_container_runtime test_container_image scc_apply_docker_image_credentials
```

Comment:
you haven't removed `clean_container_host` from export

Reply:
*this change is ready for review*